framework/src/ppl/delite/framework/ops/DeliteOpsCGen.scala

```diff
@@ -119,6 +119,7 @@ trait CGenDeliteOps extends CGenLoopsFat with GenericGenDeliteOps with CGenDelit
 
   def emitAbstractFatLoopHeader(syms: List[Sym[Any]], rhs: AbstractFatLoop) {
     val kernelName = getKernelName(syms)
+    stream.println("#include \"pcmHelper.h\"")
```

Comment:
doesn't this belong with the other header definitions in cpphelperFuncs? or why isn't is just included by DeliteCpp.h instead? @hyouklee: opinon?